artman/cli/support.py

```diff
@@ -62,54 +62,17 @@ def parse_github_credentials(config, argv_flags):
 
     # Return the appropriate credentials.
     return {
-        'username': argv_flags.github_username or config['username'],
-        'token': argv_flags.github_token or config['token'],
+        'username': argv_flags.github_username or github_config.username,
+        'token': argv_flags.github_token or github_config.token,
     }
 
 
-def parse_local_paths(user_config, root_dir):
-    """Parse all relevant local flags, given appropriate user config and user.
+def resolve(name, flags, default=None):
```

Comment:
ISTM this method is no longer necessary and can be replaced with `flags.vars().get(name, default)`